test/test_io_live.py

```diff
@@ -81,16 +84,15 @@ def do_test(self, n_ifos, n_ifos_followup):
             psd_samples = np.random.exponential(size=1024)
             psd = FrequencySeries(psd_samples, delta_f=1.)
 
-            # fill in the various fields
-            if ifo in trig_ifos:
-                base = 'foreground/' + ifo + '/'
-                results[base + 'end_time'] = t_peak + offset
-                results[base + 'snr'] = amplitude
-                results[base + 'sigmasq'] = np.random.uniform(1e6, 2e6)
+            # fill in the various fields - all ifos now get end_time, etc
+            base = 'foreground/' + ifo + '/'
+            results[base + 'end_time'] = t_peak + offset
+            results[base + 'snr'] = amplitude
+            results[base + 'sigmasq'] = np.random.uniform(1e6, 2e6)
             followup_data[ifo] = {'snr_series': snr_series,
                                   'psd': psd}
 
-        for ifo, k in itertools.product(trig_ifos, self.template):
+        for ifo, k in itertools.product(all_ifos, self.template):
```

Comment:
Please revert this change. I do want the `CandidateForGraceDB` class to support the case where we do have some detectors below threshold in addition to the ones that generated the candidate, even if this situation is not met in the current implementation of PyCBC Live.

Reply:
The current situation does, if I understood it, allow for subthreshold ifos, but these are fed to CandidateFGDB as if they were triggering ifos, ie with the same structure of SNR time series and peak 'trigger' values.
What is the alternative if ifos are to be treated differently but still used for sky localization? Their SNR series will have to be evaluated, where is that supposed to happen?

Reply:
More broadly, what should be written in the CandidateFGDB docstring to better describe what it ought to do with the different sets of ifos?
Unless we have a technical description of the desired behaviour it's hard to write a meaningful test